styles/pages/error.less

```diff
@@ -4,33 +4,36 @@
   display: flex;
   height: 100vh;
 
+  &--has-image {
+    flex-direction: column;
+    height: auto;
+    padding-top: @spacing-default*19;
```

Comment:
I dont think this has to be a variable. This value is only used once. and I think its a bit weird to use default spacing for this very large padding-top. maybe you could say it has a margin-top: auto; not sure if this has the same desired out put but you could try it.

Reply:
you could also make a custom local variable

Reply:
I recently refactored the use of spacing variables for margin/padding to only have 1 variable instead of `@spacing-s` / `@spacing-m` / `@spacing-l` for calculations and using it consistently throughout the project.

Since `@spacing-default` is always the same value. It's just a matter of multiplying that times 19 and you have an idea of what the actual value might be.

Reply:
Maybe we can make this a custom variable in the future instead of multiplying variables like *19